src/frontend/screens/Login/index.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useContext, useEffect, useState } from 'react'
 import './index.css'
 import Runner from './components/Runner'
 import { useTranslation } from 'react-i18next'
-import { useNavigate } from 'react-router'
+import { useNavigate } from 'react-router-dom'
```

Comment:
unrelated to the PR but those imports were wrong for a long time now so I don't think we need a PR for that alone.